server/bin/setup

```diff
@@ -0,0 +1,29 @@
+#! /bin/bash
+
+echo "Downloading dependencies:"
+yarn
+
+echo "Copy env variables:"
+cp ../.env.sample ../.env
+
+# DATABASE
+echo "Create database:"
+node_modules/.bin/sequelize db:create
+
+echo "Create database for test db:"
+NODE_ENV=test node_modules/.bin/sequelize db:create
+
+echo "Runing migrations:"
+node_modules/.bin/sequelize db:migrate
+
+echo "Runing migrations for test db:"
+NODE_ENV=test node_modules/.bin/sequelize db:migrate
+
+# GLOBAL modules
+echo "For interacting with sequalize cli directly from command line:"
+echo "NOTE: you can skip this step and use node_modules/.bin/sequelize 😉"
```

Comment:
@AleksSenkou how is it possible to skip this step without editing this file?

Reply:
@DDKatch just to remove lines from the file. It's oriented on advanced devs. For beginners `bin/setup` will be enough, thoughts?